gtfu/src/main/java/gtfu/tools/CheckFeedExpiration.java

```diff
@@ -0,0 +1,137 @@
+package gtfu.tools;
+
+import java.util.Date;
+
+import gtfu.ConsoleProgressObserver;
+import gtfu.Debug;
+import gtfu.EmailFailureReporter;
+import gtfu.FailureReporter;
+import gtfu.FeedInfo;
+import gtfu.ProgressObserver;
+import gtfu.Recipients;
+import gtfu.SlackFailureReporter;
+import gtfu.Time;
+import gtfu.Util;
+
+/**
+ * Daily checks and updates on each active agency's feed.
+ */
+public class CheckFeedExpiration {
+    private static final long MAX_FEED_EXPIRATION_WINDOW_DAYS = 14;
+
+     /**
+     * Runs CheckFeedExpiration for a single agency
+     * @param agencyID The agencyID
+     */
+    public static void checkFeedExpiration(String agencyID, boolean sendEmailAlert, boolean sendSlackAlert) throws Exception {
```

Comment:
A little better, but still not good. Naming a method of a class the same as that class is confusing, casing aside. I recommend just doing a regular constructor and then creating an instance from main():

```
CheckFeedExpiration cfe = new CheckFeedExpiration(/*args*/);
```

 This also allows intuitive use of this class from other code.